src/test/scala/tech/cryptonomic/conseil/tezos/InMemoryDatabase.scala

```diff
@@ -12,18 +11,31 @@ import scala.concurrent.duration._
   */
 trait InMemoryDatabase extends BeforeAndAfterAll with BeforeAndAfterEach {
   self: TestSuite =>
+  import java.nio.file._
+  import scala.collection.JavaConverters._
+  import slick.jdbc.PostgresProfile.api._
+  import ru.yandex.qatools.embed.postgresql.EmbeddedPostgres
+  import ru.yandex.qatools.embed.postgresql.distribution.Version
 
-  /** defines configuration for a randomly named h2 in-memory instance */
+  protected val databaseName = "conseil-test"
```

Comment:
I'm not sure if I like hardcoding all the information about connecting to the DB in here, as previous case had everything in one string, now if your port is taken you won't run it without changing what's hardcoded. Maybe you should consider creating config for the testing so there would be a possibility to run tests with different one, instead changing those values.

Reply:
It might make sense, I actually changed port because the default one might be taken locally from a dev's running instance, otherwise I wouldn't have cared about that anyway.

I find the idea of having a configuration specific for testing purposes a bit overboard... I'll think about it a bit more...

In general the test config parameters might be set from individual tests, on another project I simply scanned for random unused ports to run multiple instances in parallel, but I don't know if it's worth the trouble at this level... embedded postgres doesn't even work on CI!

Reply:
For the time being I'll leave it like this... open to improvements later down the road